examples/account/create_vp.rs

```diff
@@ -4,75 +4,119 @@
 //! A Verifiable Presentation (VP) represents a bundle of one or more Verifiable Credentials.
 //! This example demonstrates building and usage of VPs.
 //!
+//! Note: This example uses in-memory storage for simplicity.
+//! See `create_did` example to configure Stronghold storage.
+//!
```

Comment:
Move this to the `AccountBuilder` code for better visibility, if we really don't want to use Stronghold.

Reply:
I think adding stronghold configuration is too much noise when the examples are already quite packed.
I added the note above the `AccountBuilder` creation.